scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -631,6 +643,19 @@ protected boolean createExecution(NodeSet nodeSet, Node node, InternalJob job, I
 
     }
 
+    private void waitForTaskStartedNotification() {
+        finalizeStartingLock.lock();
+        try {
+            try {
+                finalizeStartingCondition.await();
+            } catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)